courses/free-courses-ur.md

```diff
@@ -12,3 +13,9 @@
 ### HTML and CSS
 
 * [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
+
+
+### Javascript
+
+* [Javascript Tutorials In Urdu and Hindi](https://www.youtube.com/watch?v=Lgxgm-T9cgA&list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf) - Yahoo Baba
```

Comment:
For YouTube, we prefer "playlist" links, not individual video links.

The link you provided seems different from the title.

Are you referring to this link: https://www.youtube.com/playlist?list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf

And use the original title from the resource and avoid inventing new ones.

Reply:
Working on it.